src/main/java/model/LottoRank.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public enum LottoRank {
+    FIFTH(3, new Money(5000)),
+    FOURTH(4, new Money(50000)),
+    THIRD(5, new Money(1500000)),
+    SECOND(5, new Money(30000000)),
+    FIRST(6, new Money(2000000000));
```

Comment:
```suggestion
    FIRST(6, new Money(2_000_000_000));
```
세 자리마다 언더바를 붙여주면 값이 얼마인지 파악하기 쉬워져요